spec/interactors/notes/assign_attributes_spec.rb

```diff
@@ -0,0 +1,31 @@
+describe Notes::AssignAttributes do
+  subject(:context) { described_class.call(note: note, note_params: params) }
+
+  let(:user) { create(:user) }
+  let(:note) { create(:note, user_id: user.id) }
+  let(:params) { attributes_for(:note, user_id: user.id) }
```

Comment:
I prefer to prescribe attributes explicitly for clarity
Also, I think it is not necessary to test all attributes for emptiness because in this case, we test validations presence. If we want to test validations, we should do it in the model spec. In this case, we only should test that if an object is invalid, interactor will raise an error.

Also, we shouldn't wrap default case into context
```ruby
let(:params) do 
  {
    title: "Lorem ipsum",
    content: "Dolor sit amet",
    user_id: user.id
  }
end

it "assigns attributes" do
  expect(context).to be_success
  expect(context.user).to have_attributes(params)
end

context "with the empty title" do
  {
    title: "",
    content: "Dolor sit amet",
    user_id: user.id
  }

  it "raises error" do
    expect(context).to be_failure
    expect(context.error).to eq "..."
  end
end
```